config/reports.json

```diff
@@ -299,19 +299,19 @@
 			}
 		},
 		"ttci": {
-			"name": "Time to Consistently Interactive",
+			"name": "Time to Interactive",
 			"type": "seconds",
 			"singular": "second",
-			"description": "The number of seconds from the time the navigation started until the page is fully interactive and the network is idle. This metric comes from [Lighthouse](https://developers.google.com/web/tools/lighthouse/audits/consistently-interactive) as is only available in mobile tests.",
+			"description": "The number of seconds from the time the navigation started until the last long task happened before 5 seconds window where network is mostly quiet. [Read more](https://docs.google.com/document/d/1GGiI9-7KeY3TPqS3YT271upUVimo-XiL5mwWorDUD4c/edit#heading=h.iqlwzaf6lqrh).",
 			"histogram": {
 				"minDate": "2017_06_01"
 			}
 		},
 		"ttfi": {
-			"name": "Time to First Interactive",
+			"name": "First CPU Idle",
 			"type": "seconds",
 			"singular": "second",
-			"description": "The number of seconds from the time the navigation started until the layout is stabilized, web fonts are visible, and the page is responsive to user input. This metric comes from [Lighthouse](https://developers.google.com/web/tools/lighthouse/audits/consistently-interactive) as is only available in mobile tests.",
+			"description": "The number of seconds from the time the navigation started until the minimally page interactive. [Read more](https://docs.google.com/document/d/1GGiI9-7KeY3TPqS3YT271upUVimo-XiL5mwWorDUD4c/edit#heading=h.d06i5ycfxaow).",
```

Comment:
It's kinda mix of prev description and https://developers.google.com/web/tools/lighthouse/audits/first-cpu-idle, because of not clear definition to copy-paste.